guides/components/index.md

```diff
@@ -174,50 +168,43 @@ form > button {
+![capture d'écran de l'app de messagerie avec le style](/images/ember-core-concepts/messaging-app-1.png)
 
-You start building parts of an Ember application using HTML, so if you already
-know HTML and CSS, you know how to build a basic Ember application!
+La construction d'une application Ember commence par l'utilisation de HTML, donc si vous connaissez déjà le HTML et le CSS, alors vous savez déjà construire une application Ember basique&nbsp;!
 
-You can even use SVG or web components without any changes. As long as your HTML
-is valid, Ember will render it.
+Vous pouvez même utiliser des SVG ou des _web components_ tels quels. Tant que votre HTML est valide, Ember l'affiche.
 
-# Self-Closing Tags
+# Balises auto-fermantes
 
-In addition to normal HTML syntax, Ember allows you to use self-closing syntax
-(`<div />`) as a shorthand for an opening and closing tag (`<div></div>`).
+En plus de la syntaxe HTML classique, Ember permet l'utilisation d'une syntaxe auto-fermante (`<div />`) comme raccourci pour les balises ouvrantes et fermantes (`<div></div>`).
 
 <div class="cta">
   <div class="cta-note">
     <div class="cta-note-body">
-      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-heading">Zoey dit...</div>
       <div class="cta-note-message">
-        You don't <strong>need</strong> to use this syntax for <a href="https://html.spec.whatwg.org/multipage/syntax.html#void-elements">"void" HTML
-        tags</a> such as <code>img</code> or <code>br</code>, which are already
-        defined as self-closing by the HTML specification, but you <strong>can</strong> use this syntax
-        as a shorthand for tags that are not self-closing.
+        Vous n'avez pas <strong>besoin</strong> d'utiliser cette syntaxe pour les <a href="https://html.spec.whatwg.org/multipage/syntax.html#void-elements">balises HTML dites "vides"</a> comme <code>img</code> ou <code>br</code>, car elles sont déjà définies comme auto-fermantes par les spécifications HTML. En revanche, vous pouvez utiliser cette syntaxe comme raccourci pour les balises qui ne sont pas auto-fermantes par définition.
       </div>
     </div>
     <img src="/images/mascots/zoey.png" role="presentation" alt="">
   </div>
 </div>
 
-# Supported Features
+# Fonctionnalités supportées
 
-This means that all of the following HTML features work as-is:
+Toute les fonctionnalités suivantes de HTML fonctionnent telles quelles&nbsp;:
 
-- Web components
+- _Web components_
 - SVG
-- HTML comments
-- White space (following the same rules as normal HTML)
-- Special HTML elements like `<table>` and `<select>`
+- Commentaires HTML
+- Espaces (les espaces suivent les mêmes règles que dans du HTML classique)
+- Éléments HTML spéciaux comme `<table>` ou `<select>`
 
 # Restrictions
 
-There are a handful of restrictions on the HTML that you can put in an Ember
-template:
+Il existe une poignée de restrictions sur le HTML pouvant être écrit dans un _template_ Ember&nbsp;:
```

Comment:
Fixed version:

```suggestion
Il existe une poignée de restrictions sur le HTML pouvant être écrites dans un _template_ Ember&nbsp;:
```

However I find this version better and less weird to read than "des restrictions qui peuvent être écrites":

```suggestion
Il existe une poignée de restrictions sur le HTML pouvant être utilisées dans un _template_ Ember :
```

Reply:
La manière dont je comprends cette phrase, le verbe se réfère au HTML, pas au restrictions :
=== Le HTML pouvant être écrit dans un template est soumis à une poignée de restrictions.